src/main/java/teammates/ui/webapi/output/InstructorsData.java

```diff
@@ -10,7 +10,11 @@
  */
 public class InstructorsData extends ApiOutput {
 
-    private final List<InstructorData> instructors;
+    private List<InstructorData> instructors;
+
+    public InstructorsData() {
+        // Use this constructor if InstructorData should be sent with the key
+    }
```

Comment:
Similar comments to https://github.com/TEAMMATES/teammates/pull/9990/files#r403644820